express-graphql/dist/renderGraphiQL.js

```diff
@@ -84,7 +84,7 @@ const introductionDemoQuery = `# Welcome to 1BahnQL
 #   }
 # }
 #
-# Querys stations and operationLocations (Betriebsstellen) matching the searchterm "Flughafen"
+# Queries stations and operationLocations (Betriebsstellen) matching the searchterm "Flughafen"
```

Comment:
Queries stations and operationLocations (Betriebsstellen) matching the **search term** "Flughafen"